Hunting Queries/SigninLogs/SpikeInFailedSignInAttempts.yaml

```diff
@@ -28,7 +29,14 @@ query: |
   TimeSeriesAlerts
     | join kind=inner (
     SigninLogs
+    | where TimeGenerated between (auditLookback..endtime)
     | summarize ResultTypeCount=count(),ResultTypes=make_set(ResultType), Locations=make_set(Location), Apps=make_set(AppDisplayName), Ips=make_set( IPAddress) by UserPrincipalName, bin(TimeGenerated, 1h)
     ) on UserPrincipalName, TimeGenerated
     | summarize AnomolyTimes = make_set(TimeGenerated), Ips = make_set(Ips), Apps = make_set(Apps), sum(anomalies), Locations=make_set(Locations) by UserPrincipalName
-    | sort by sum_anomalies desc
\ No newline at end of file
+    | sort by sum_anomalies desc
+    | extend timestamp = tostring(AnomolyTimes[0]), AccountCustomEntity = UserPrincipalName
+entityMappings:
+  - entityType: Account
+    fieldMappings:
+      - identifier: AadUserId
```

Comment:
identifier should be FullName instead of AadUserId based on ColumnName: UserPrincipalName